lib/constants.js

```diff
@@ -120,6 +120,11 @@ export const LANGUAGES = [
     module: 'dockerfile'
   },
   {
+    name: 'Elixir',
+    module: 'elixir',
```

Comment:
As of #58, we've renamed `module` -> `mode`. Other than that, this looks good. Thanks!